aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageRemoveOperation.java

```diff
@@ -73,6 +75,7 @@ public AWSS3StorageRemoveOperation(
         this.awsS3StoragePluginConfiguration = awsS3StoragePluginConfiguration;
     }
 
+    @OptIn(markerClass = InternalAmplifyApi.class)
```

Comment:
Is it possible to opt-in via compiler flags for Java similarly to how we do it for Kotlin?

Reply:
I haven't been able to find any references that would support this.